exercises/fetch_json/problem.md

```diff
@@ -30,7 +20,4 @@ There are several things you will want to know:
 2. Parse the returned JSON and `console.log` it for much win.
 
 This challenge is a bit tricky but the implementation is relatively
-straightforward.  If you get stuck, refer to the `q-io` documentation for
-clarification:
-
-  https://github.com/kriskowal/q-io
+straightforward.
```

Comment:
Maybe. It uses bluebird under the hood, but as long as it's all abstracted from the user, does that matter?